subscription-offer-prompt/index.js

```diff
@@ -65,5 +66,23 @@ function createSubscriptionPrompt (html) {
 	};
 }
 
-export const initPrompt = executeWhen(createSubscriptionPrompt(createPopupHTML));
+function getSubscriptionPromptValues () {
+	const currencyCode = 'GBP';
```

Comment:
This needs to be configured to the user's currency code. I don't know how to get or derive this. 

@matthew-andrews @wheresrhys @ironsidevsquincy @ifyio advice?